docs/examples/client.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Node packages
+ */
+const didCommon = require('@decentralized-identity/did-common-typescript');
+const didAuth = require('@decentralized-identity/did-auth-jose');
+
+/**
+ * Constants
+ */
+const discoveryEndpoint = 'HTTP_RESOLVER_ENDPOINT_HERE'; // e.g. https://beta.discover.did.microsoft.com/
+const redirectUrl = 'exampleServerUrl.com'; // server url that client will send Authentication Response to
+
+/**
+ * Fill in the client DID to use
+ */
+const clientDID = 'did:example:YOUR_Client_DID_HERE';
```

Comment:
The entire thing is the DID. It may be confusing to say the DID goes after beginning of an example DID and lead to people substituting `did:example:did:foo:bar`
```suggestion
const clientDID = 'YOUR_Client_DID_HERE';
```